packages/react-core/src/components/Nav/examples/Nav.md

```diff
@@ -55,6 +55,8 @@ class NavDefaultList extends React.Component {
 
 ### Grouped
 
+Note: to keep nav groups accessible an aria label should be supplied if a title prop is not passed.
```

Comment:
I'm not sure if there is a convention for referring to attrs/props, but it might be good to write it this way?

```suggestion
Note: to keep nav groups accessible an `aria-label` should be supplied if the `title` prop is not passed.
```